### PR TITLE
Ts core test

### PR DESCRIPTION
**Pull request by tbosch**

### CODE REVIEW

File: modules/angular2/src/test_lib/utils.ts [hunk 1]
```diff
@@ -41,3 +41,12 @@ export function dispatchEvent(element, eventType) {
 export function el(html: string) {
   return DOM.firstChild(DOM.content(DOM.createTemplate(html)));
 }
+
+var _ESCAPE_RE = RegExpWrapper.create('[\\-\\[\\]\\/\\{\\}\\(\\)\\*\\+\\?\\.\\\\\\^\\$\\|]');
```
Comment (rkirov) [inline modules/angular2/src/test_lib/utils.ts]: what a monster, add a comment what exactly it matches.

Comment (gdi2290) [inline modules/angular2/src/test_lib/utils.ts]: also a link to https://regex101.com/ would be awesome

File: modules/angular2/src/test_lib/utils.ts [hunk 2]
```diff
@@ -41,3 +41,12 @@ export function dispatchEvent(element, eventType) {
...
+function escapeRegexpStr(input: string): string {
+  return StringWrapper.replaceAllMapped(input, _ESCAPE_RE, (match) => `\\${match[0]}`);
+}
+
+export function containsRegexp(containsStr: string): RegExp {
+  return RegExpWrapper.create(`.*${escapeRegexpStr(containsStr)}.*`);
```
Comment (rkirov) [inline modules/angular2/src/test_lib/utils.ts]: not sure why you need this since `/b/.test("abc") -> true`

Comment (tbosch) [inline modules/angular2/src/test_lib/utils.ts]: You are right. Will remove the prefix / suffix.

**Pull request merged**